### PR TITLE
Improving build process fixes #29

### LINKED ISSUE #29: Improve Build Process
@SkycoinSynth requested multiple changes to the build process

1. Print the Sha256 of the firmware binary after the build process. 

2. Print the Sha256 of the bootloader after the build process. 

3. Put the binaries created into a build directory rather than root. 

4. Print the relative path from root where the binaries were put.

### PR DESCRIPTION
**Pull request by Midren**
Fixes #29 

 Changes:	
- Refactored prepare.py and also add additional verbose info like the location of binary files and sha256

 Does this change need to mentioned in CHANGELOG.md?	
no	

 Requires testing	
no

### CODE REVIEW

File: ci-scripts/compute_hash.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import hashlib
+import binascii
+import argparse
+
+
+def calculate_binary_hash(path):
+    with open(path, "rb") as f:
+        binary = f.read()
+        hex_hash = hashlib.sha256(binary).hexdigest()
+        return "".join(['\\x' + hex_hash[i:i + 2] for i in range(0, len(hex_hash), 2)])
```
Comment (jdknives) [inline ci-scripts/compute_hash.py]: Why are you returning the hash in this peculiar form?

Comment (Midren) [reply ci-scripts/compute_hash.py]: So then it could be easily copy-pasted in code.

**Comment by Midren** [comment]
@jdknives Now after building full-firmware you will get next message:

> 
> Bootloader's location   : build/bootloader-no-memory-protect.bin
> Firmware's location     : build/skyfirmware.bin
> Full-firmware's location: releases/full-firmware-no-memory-protect.bin
> 
> bootloader : 32768 bytes
> firmware   : 176364 bytes
> combined   : 241644 bytes
> 
> bootloader's SHA256: dd4cfcdaa5a15c152d03173067213829989b65eab5870aace407098f6bcd0dff 
> firmware's SHA256  : 2688a7854e8581b8c4082225031c8bea765f95f03dcb71834d003ddce48d46ec

Is it okay, or should be changed somehow ?

**Comment by jdknives** [comment]
@RomanMilishchuk that seems alright. Please capitalize Bootloder, Firmware and Combined and then this is good to go.

**Review by jdknives — APPROVED** [review]

**Pull request merged**